.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,107 @@
+name: Bug Report
```

Comment:
Minor nitpick -- this was previously "Bug **r**eport" (matching "Enhancement request", "Documentation issue", etc)